### PR TITLE
Adding allocated su available and normalization factor column to resource specs.

### PR DESCRIPTION
**Pull request by eiffel777**
This adds an `allocated_su_availabe` column to `modw_aggregates.resourcespecsfact_by` aggregate tables. This column takes into account what percent of a machine is allocated to be used. The statistics that use this column will be added to the xdmod-xsede module.

It also adds a normalization factor column for the resource specs realm. This column can be used to help compare the capacity of different resources.

## Tests performed
Tested in docker

## Checklist:

- [ ] The pull request description is suitable for a Changelog entry
- [x] The milestone is set correctly on the pull request
- [x] The appropriate labels have been added to the pull request

### CODE REVIEW

**Review by aestoltm — APPROVED** [review]

File: configuration/etl/etl_tables.d/jobs/xdw/resource-specs.json [hunk 1]
```diff
@@ -75,6 +75,12 @@
                 "nullable": true,
                 "comment": "Total number of SU's available per day"
             },
+            {
+                "name": "normalization_factor",
+                "type": "int(10) unsigned",
```
Comment (aaronweeden) [inline configuration/etl/etl_tables.d/jobs/xdw/resource-specs.json:80]: Is it always going to be an integer?

File: configuration/etl/etl_tables.d/common/staging/resource-spec.json [hunk 2]
```diff
@@ -80,6 +80,13 @@
                 "nullable": true,
                 "comment": "Total number of SU's available per day"
             },
+            {
+                "name": "normalization_factor",
+                "type": "int(10) unsigned",
+                "nullable": true,
+                "default": 1,
+                "comment": "Nomralization factor for a resource. This helps compare capacity across resources"
```
Comment (aaronweeden) [inline configuration/etl/etl_tables.d/common/staging/resource-spec.json:88]:
```suggestion
                "comment": "Normalization factor for a resource. This helps compare capacity across resources"
```

File: configuration/etl/etl_tables.d/jobs/xdw/resource-specs.json (hunk 1 above)
Comment (aaronweeden) [reply configuration/etl/etl_tables.d/jobs/xdw/resource-specs.json:80]: Resolved in https://github.com/ubccr/xdmod/pull/1836

File: configuration/etl/etl_tables.d/common/staging/resource-spec.json (hunk 2 above)
Comment (aaronweeden) [reply configuration/etl/etl_tables.d/common/staging/resource-spec.json:88]: Resolved in https://github.com/ubccr/xdmod/pull/1836

**Pull request merged**